analyzers/src/SonarAnalyzer.Common/AnalysisContext/DiagnosticAnalyzerContextHelper.cs

```diff
@@ -22,7 +22,7 @@
 using System.Runtime.CompilerServices;
 using Microsoft.CodeAnalysis.Text;
 
-namespace SonarAnalyzer.Helpers
+namespace SonarAnalyzer
```

Comment:
This should respect the folder structure and be named `SonarAnalyzer.AnalysisContext`. Same for the others.

Reply:
I've added FIXME in later PR to deal with this. As `AnalysisContext` happens to be a Roslyn class too, it causes compilation errors like "namespace is used as a class"